numpy/random/tests/test_random.py

```diff
@@ -13,19 +13,19 @@
 class TestSeed(TestCase):
     def test_scalar(self):
         s = np.random.RandomState(0)
-        assert_equal(s.randint(1000), 684)
```

Comment:
Leave the tests the way they were, and add new ones for the new functionality. One for each of the cases

  - no args
  - low=None, high=None
  - low>0, high=None
  - low=0, high=None
  - low<0, high=None
  - low=None, high < 0
  - low=None, high=0
  - low=None, high > 0